src/gui/torrentcontentmodel.cpp

```diff
@@ -50,6 +53,14 @@
 #include "torrentcontentmodelfolder.h"
 #include "torrentcontentmodelfile.h"
 
+#ifdef Q_OS_MAC
+struct NSImage;
+// This function is a private QtGui library export on macOS
+// See src/gui/painting/qcoregraphics_p.h for more details
+// QtMac::fromCGImageRef takes a CGImageRef and thus requires a double conversion
+QPixmap qt_mac_toQPixmap(const NSImage *image, const QSizeF &size);
```

Comment:
Do we have a reasonable alternative? I think not. So IMO, this specific case is ok?